linode/manager
# feat: [M3-8185] - Add Accessibility tab to Storybook Stories

## Description 📝
We could utilize Storybook more. One way we could do that is by using it for Accessibility testing when developing a component

## Changes  🔄
- Add Accessibility tab to Stories

## Preview 📷
https://github.com/user-attachments/assets/a61c8030-63f0-4a0c-8524-30bb841b8253

## How to test 🧪
### Verification steps
(How to verify changes)
- Pull this PR and run storybook locally
- Open a Story and you should see a new Accessibility tab

## As an Author I have considered 🤔

*Check all that apply*
